config/brakeman.ignore

```
@@ -44,6 +44,40 @@
       ],
       "note": ""
     },
+    {
```

Instead of adding brakeman.ignore exception, insert indicated parameter into h()